backend/src/schema/resolvers/rooms.ts

```diff
@@ -44,7 +29,14 @@ export default {
           ON CREATE SET
             room.createdAt = toString(datetime()),
             room.id = apoc.create.uuid()
-          RETURN room { .* }
+          WITH room, user, currentUser,
+          user.name AS roomName
+          RETURN room {
+            .*,
+            users: [properties(currentUser), properties(user)],
+            roomName: roomName,
+            unreadCount: toString(0)
```

Comment:
This `toString(0)` is not necessarily true, since you can (and will) call `createRoom` even for an existing room, which might contain unread messages for the user.

As I said before - the createRoom call has downsides (empty rooms), but for Implementation its easier. The rooms are queried after `createRoom` is called so we can certainly live with this little potential inaccuracy.

Reply:
We need `toString` because as java number object is returned: `{ high: 0, low: 5 }. Javascript can not handle this object